chap-architecture.tex

```diff
@@ -3199,31 +3199,12 @@ \section{Potential Future Changes to the CHERI Architecture}
   represents, and the caller might want to restrict these permissions before
   passing the sealed capability to another subsystem.
 
-\item
```

Comment:
Even in a split world you'd just CIncOffset cd, cnull, int...

Reply:
Yeah I'm pretty sure this text predates the null register

Reply:
I wonder if we want any rationale chapter words on the NULL capability ..

Reply:
Maybe.  Note that the x86 sketch does not yet have a null capability.  This is because x86 presumes a merged register file.  RISC-V's cnull is a pragmatic extension of the existing zero integer register (x0).  It's not clear to me that cnull is an architecture-independent requirement.